downstream/modules/platform/ref-platform-components.adoc

```diff
@@ -0,0 +1,50 @@
+[id="ref-platform-components"]
+
+= {PlatformName} platform components
+
+Red Hat Ansible Automation Platform consists of the following components:
+
+.{HubNameMain}
+A repository for certified content of Ansible Content Collections. 
+{HubNamemain} is the centralized repository for Red Hat and its partners to publish content, and for customers to discover certified, supported Ansible Content Collections. 
+{CertifiedCon} provides users with content that has been tested and is backed by Red Hat.
+ 
+.{PrivateHubNameStart}
+If you create your own Ansible content, the Ansible private automation hub provides a way to publish and collaborate on it within your organization behind your firewall or on disconnected systems. 
+You can also add to it curated content from {HubNameMain} and {Galaxy}.
+
+ 
+.{ControllerNameStart}
+An enterprise framework for controlling, securing, and managing Ansible automation with a user interface (UI) and RESTful application programming interface (API).
+ 
+.{CatalogNameStart}
+{CatalogNameStart} is a service within {PlatformName}. 
+{CatalogNameStart} enables you to organize and govern product catalog sources on Ansible {ControllerName} across various environments.
+
+Using automation services catalog you can:
+
+* Apply multi-level approval to individual platform inventories.
+* Organize content in the form of products from your platforms into portfolios.
+* Choose portfolios to share with specific groups of users.
+* Set boundaries around values driving execution of user requests.
+ 
+.{AutomationMeshStart}
+{AutomationMeshStart} is an overlay network intended to ease the distribution of work across a large and dispersed collection of workers through nodes that establish peer-to-peer connections with each other using existing networks.
+
+{AutomationMeshStart} provides:
+
+* Dynamic cluster capacity that scales independently, allowing you to create, register, group, ungroup and deregister nodes with minimal downtime.
+* Control and execution plane separation that enables you to scale playbook execution capacity independently from control plane capacity.
+* Deployment choices that are resilient to latency, reconfigurable without outage, and that dynamically re-reroute to choose a different path when outages may exist. mesh routing changes.
```

Comment:
```suggestion
* Deployment choices that are resilient to latency, reconfigurable without outage, and that dynamically re-reroute to choose a different path when outages may exist. Mesh routing changes.
```
"Mesh routing changes." seems like an odd short sentence to add at the end. I would remove it since it seems to be redundant. This edit it up to you.